week5/README.md

```diff
@@ -0,0 +1,52 @@
+# Week 5 Homework
+
+## Part 1 - Short Answer Questions
+
+### a) _________ are used to customize the appearance and behavior of a View.
+- Modifiers
+
+### b) The basic building block of a SwiftUI interface is a ______________.
+- View
+
+### c) Each View in SwiftUI is a ___________ type.
+- Struct
+
+### d) List two ways you can find the modifiers of a View.
+1. Inspecting the code in Xcode
+2. Using Xcode's SwiftUI Inspector
+
+### e) True or False: A modifier changes a View.
+- True
```

Comment:
Maybe yes from one point of view, but technically no. Modifiers return a new view that wraps the original view and replaces it in the view hierarchy.